client/src/components/auth/login/Login.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import axios from 'axios'
+import classnames from 'classnames'
+
+const Form = styled.form`
+  display: grid;
+  grid-gap: 2px;
+  grid-template-rows: auto;
+`
+
+const Input = styled.input`
+  height: 48px;
+  padding-left: 10px;
+  border-radius: 4px;
+  border: 2px solid lightgray;
+
+  &.is-valid {
+    border-radius: 4px;
+    border: 2px solid lightblue;
+  }
+
+  &.is-invalid {
+    border-radius: 4px;
+    border: 2px solid red;
+  }
+`
+
+const Button = styled.button`
+  height: 48px;
+  background: papayawhip;
+  color: steelblue;
+  font-size: 20px;
+  border: 3px solid steelblue;
+  border-radius: 3px;
+  border-radius: 4px;
+`
+
+const Validation = styled.div`
+  font-family: system-ui;
+  padding-left: 10px;
+  font-size: 12px;
+  color: gray;
+`
+
+export default function Login() {
+  const [loginData, setLoginData] = useState({
+    email: '',
+    password: '',
+    errors: {},
+  })
+
+  function onChange(e) {
+    setLoginData({ ...loginData, [e.target.name]: e.target.value })
+  }
+
+  function onSubmit(e) {
+    e.preventDefault()
+
+    const user = {
+      email: loginData.email,
+      password: loginData.password,
+    }
+    console.log(user)
+    axios
+      .post('/users/login', user)
+      .then(res => console.log(res.data))
```

Comment:
Try `this.props.history.push('/')` and wrap the component with `withRouter` or otherwise use `window.location.assign('/')`